lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -192,6 +212,10 @@ function dosomething_northstar_transform_user($user, $password = null) {
 
   // Set values in ns_user if they are set.
   foreach ($optional as $ns_key => $drupal_key) {
+    if (! isset($user->$drupal_key)) {
```

Comment:
@DFurnes noticed this has been resulting in errors 
![screen shot 2016-09-09 at 10 34 53 am](https://cloud.githubusercontent.com/assets/897368/18390696/5298f506-7679-11e6-898a-88602d4c345b.png)

Mind if I go back to the way I had it? I know it was an extra conditional but it prevented the warnings from popping up